src/main/java/frc/robot/subsystems/gripper/Gripper.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot.subsystems.gripper;
+
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import frc.robot.Ports;
+import frc.robot.subsystems.LoggedSubsystem;
+
+public class Gripper extends LoggedSubsystem<GripperLoggedInputs> {
+    private final Solenoid leftSolenoid = new Solenoid(PneumaticsModuleType.CTREPCM, Ports.Gripper.LEFT_SOLENOID);
+    private final Solenoid rightSolenoid = new Solenoid(PneumaticsModuleType.CTREPCM, Ports.Gripper.RIGHT_SOLENOID);
+
+
+    public Gripper(GripperLoggedInputs inputs) {
```

Comment:
Make this constructor private and add `getInstance`

Reply:
done.